modules/imgcodecs/src/grfmt_jpeg2000_openjpeg.hpp

```diff
@@ -61,13 +61,12 @@ using ImagePtr = std::unique_ptr<opj_image_t, detail::OpjImageDeleter>;
 
 } // namespace detail
 
-class Jpeg2KOpjDecoder CV_FINAL : public BaseImageDecoder
+class Jpeg2KOpjDecoderBase : public BaseImageDecoder
```

Comment:
I think it is better to move this class to `detail` namespace - it is not supposed to be instantiated directly

Reply:
It is not a part of public headers, so it is not available externally.
So, `detail` namespace is not necessary here.

Reply:
I've just moved it to detail. Should I roll back the change then?

Reply:
As you wish. Both cases should be OK.

Reply:
Let's leave it like it is now.